python/hsfs/engine/python.py

```diff
@@ -946,15 +946,15 @@ def acked(err, msg):
             feature_group.backfill_job.run(
                 args=feature_group.backfill_job.config.get("defaultArgs", "")
                 + " -kafkaOffsetReset true",
-                await_termination=offline_write_options.get("wait_for_job", True),
+                await_termination=offline_write_options.get("wait_for_job", False),
```

Comment:
Shouldn't changes be also made here? https://github.com/logicalclocks/feature-store-api/pull/995/files#diff-e77e0390cdce0c2b0a51916239c6a232a59f58974c38edd465d1859fc5c53d48R76

Reply:
we don't want to wait for all jobs, just the hudi_fg.insert() jobs specifically.